pipeline.puml

```diff
@@ -0,0 +1,19 @@
+@startuml
```

Comment:
This should probably be moved to the documentation

Reply:
done